### PR TITLE
Remove _get_removed_version_from_deprecated_version function

### PR DESCRIPTION
**Pull request by nuka137**
## Motivation

See #2902

## Description of the changes

Remove `_get_removed_version_from_deprecated_version` function.

NOTE: This PR must be rebased after merging #3064

### CODE REVIEW

File: optuna/integration/keras.py [hunk 1]
```diff
@@ -12,18 +12,17 @@
...
     Callback = object  # NOQA
 
+_keras_pruning_callback_deprecated_msg = "Recent Keras release (2.4.0) simply redirects "
+"all APIs in the standalone keras package to point to tf.keras. "
+"There is now only one Keras: tf.keras. "
+"There may be some breaking changes for some workflows by upgrading to keras 2.4.0. "
+"Test before upgrading. "
+"REF: https://github.com/keras-team/keras/releases/tag/2.4.0. "
+"There is an alternative callback function that can be used instead: "
+":class:`~optuna.integration.TFKerasPruningCallback`"
```
Comment (himkt) [inline optuna/integration/keras.py:22]:
```suggestion
_keras_pruning_callback_deprecated_msg = (
    "Recent Keras release (2.4.0) simply redirects "
    "all APIs in the standalone keras package to point to tf.keras. "
    "There is now only one Keras: tf.keras. "
    "There may be some breaking changes for some workflows by upgrading to keras 2.4.0. "
    "Test before upgrading. "
    "REF: https://github.com/keras-team/keras/releases/tag/2.4.0. "
    "There is an alternative callback function that can be used instead: "
    ":class:`~optuna.integration.TFKerasPruningCallback`"
)
```

**Comment by himkt** [comment]
@nuka137 Thank you for the PR. I left one minor comment. And please fix type checking error raised by mypy. :pray:

**Comment by nuka137** [comment]
@himkt 

Thanks for reviewing.
I think this PR needs to rebase after #3064 is merged because of the duplication of changes.
I will fix all issues and remove WIP status ASAP when #3064 is merged.

File: optuna/integration/keras.py (hunk 1 above)
Comment (himkt) [reply optuna/integration/keras.py:22]: It solves in #3064.

**Comment by himkt** [comment]
Sorry for the duplicated suggestion. :bow: Yes, you're right and please rebase #3064 after merging it.

**Comment by nuka137** [comment]
@himkt 

Thanks for waiting my work.
It seems that #3064 was merged, I rebased this branch with some fixes and removed WIP status.

**Comment by toshihikoyanase** [comment]
@nuka137 I'm sorry for the delayed assignment of reviewers.

@himkt Could you continue the review, please?
@HideakiImamura Could you review this PR, please? This PR seems to be a follow-up of #3064.

**Review by himkt — APPROVED** [review]
Sorry @nuka137 for the delayed review. Thank you for improving the visibility of deprecated methods/classes. LGTM!

**Review by HideakiImamura — APPROVED** [review]
LGTM!

**Pull request merged**